include/micro_ros_utilities/type_utilities.h

```diff
@@ -32,18 +32,61 @@ extern "C"
 // Memory rule
 typedef struct micro_ros_utilities_memory_rule_t
 {
+  /**
+   * Pattern matching the fully qualified name of a msg field.
+   *
+   */
   const char * rule;
+
+  /**
+   * Memory (in bytes) to allocate for fields matching \ref rule.
```

Comment:
```suggestion
   * Maximum string or sequence capacity for fields matching \ref rule.
```

Reply:
As a developer I'd like to get some feeling for what the effects are when setting `size` to a certain value. And it would be good to be as clear as possible here.

Can we way anything about the memory use either of these two cases would result?

Even if that's dependent on the size of the element(s), we could refer to that.

Reply:
I think its clear enough, as the developer should be familiar with the concept `sequence`.
Anyway, feel free to propose a change.